BetterStacking/Patches.cs

```diff
@@ -1,145 +1,191 @@
-                {
-                    __result = null;
-                    return false;
-                }
-
-                Implementation.MergeIntoStack(normalizedCondition, numUnits, targetStack);
-                __result = targetStack;
-                return false;
-            }
-        }
+        "GEAR_BirchSaplingDried",
+        "GEAR_BearHideDried",
+        "GEAR_BottleAntibiotics",
+        "GEAR_BottlePainKillers",
+        "GEAR_CoffeeTin",
+        "GEAR_GreenTeaPackage",
+        "GEAR_GutDried",
+        "GEAR_LeatherDried",
+        "GEAR_LeatherHideDried",
+        "GEAR_MapleSaplingDried",
+        "GEAR_MooseHideDried",
+        "GEAR_PackMatches",
+        "GEAR_RabbitPeltDried",
+        "GEAR_WolfPeltDried",
+        "GEAR_WoodMatches",
+        };
+
+        private static bool PostFixTrack { get; set; } = false;
+
+        private static float PostfixCondition { get; set; } = 0;
+
+        private static GearItem? PostfixStack { get; set; } = null;
+
+        private static GearItem? PostfixGearToAdd { get; set; } = null;
+
+        private static float PostfixConstraint { get; set; } = 0;
+
+        [HarmonyPatch]
+        internal static class PlayerManager_TryAddToExistingStackable
+        {
+
+            static MethodBase? TargetMethod()
+            {
+                MethodInfo? targetMethod = typeof(PlayerManager)
+                    .GetMethods()
+                    .FirstOrDefault(
+                        m => m.Name == nameof(PlayerManager.TryAddToExistingStackable)
+                        && m.ReturnType == typeof(bool)
+                        && !m.IsGenericMethod
+                        && m.GetParameters().Length == 4
+                        );
+                if (targetMethod is null)
+                {
+                    Implementation.LogWarning("PlayerManager.TryAddToExistingStackable not found for patch.");
+                }
+
+                return targetMethod;
+            }
+
+            internal static bool Prefix(ref GearItem gearToAdd, float normalizedCondition, int numUnits, ref GearItem existingGearItem)
+            {
+                if (normalizedCondition == 0)
+                {
+                    return false;
+                }
+
+                if (UseDefaultStacking(gearToAdd))
+                {
+                    return true;
+                }
+
+                GearItem targetStack = GameManager.GetInventoryComponent().GetClosestMatchStackable(gearToAdd.GearItemData, normalizedCondition);
+
+                if (!CanBeMerged(targetStack, gearToAdd))
+                {
+                    return false;
+                }
+
+                if (targetStack.m_StackableItem != null)
+                {
+                    MergeIntoStack(normalizedCondition, numUnits, targetStack, gearToAdd);
+                    existingGearItem = targetStack;
+                    return true;
+                }
+
+                return false;

```

Comment:
Same as the rest of the method. I'm confused about "skipping vs not skipping" and ensuring that out parameters get set whenever the original is not run.